backoffice/workflows/api/serializers.py

```diff
@@ -21,3 +21,13 @@ class WorkflowDocumentSerializer(DocumentSerializer):
     class Meta:
         document = WorkflowDocument
         fields = "__all__"
+
+
+class AuthorResolutionSerializer(serializers.Serializer):
+
+    RESOLUTION_CHOICES = [
```

Comment:
they dont seem to be entirely the same, 
on the author resolution we have "accept, and "reject" which are two possible states within the `approval` stage

I will move `RESOLUTION_CHOICES` to constants so it may be reused in other places later on